index.d.ts

```diff
@@ -0,0 +1,12 @@
+interface Dictionary<T = any> {
```

Comment:
```suggestion
interface Dictionary<T = unknown> {
```
any should never be used, since it defeats the entire purpose of having a type system